custom_components/ideal_led/idealled.py

```diff
@@ -299,14 +317,13 @@ async def set_effect(self, effect: str, brightness: int | None = NotImplemented)
         self._color_mode = None
         effect_id = EFFECT_MAP.get(effect)
         if effect_id > 11: effect = 11
-        brightness_pct = int(brightness * 100 / 255)
+        brightness_pct = int(brightness /255 * 100)
         packet = bytearray.fromhex("0A 4D 55 4C 54 08 00 64 50 07 32 00 00 00 00 00")
         packet[5]  = effect_id
         packet[6]  = 0 # reverse
         packet[8]  = 50 # speed
-        packet[10] = 100 # brightness_pct # 50 # saturation (brightness?)
+        packet[10] = brightness_pct # 2024-11-16 Was: 100 # This was here before: brightness_pct # 50 # saturation (brightness?)
```

Comment:
Packet 10 seems to be the brightness of the RGB colours.  If an LED is set as white, this byte doesn't effect the brightness.  This is still an improvement over what we had before.